### PR TITLE
Show correct message when uploaded file is too large

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27717.

### Summary of Changes

Corrects error message when uploading too large files in `com_media`.

### Testing Instructions

1. Try to upload files larger than your system allows.
2. Try to upload unsafe files (e.g. PHP files).
3. Try to upload small and safe files (e.g. images).

### Expected result

1. Upload fails with message: `Total size of upload exceeds the limit.`
2. Upload fails with message: `You have tried to upload file(s) that are not safe.`
3. File is uploaded successfully.

### Actual result

1. Upload fails with message: `You have tried to upload file(s) that are not safe.`
2. Upload fails with message: `You have tried to upload file(s) that are not safe.`
3. File is uploaded successfully.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by richard67** [comment]
@SharkyKZ With this PR applied and trying to upload a PHP file I get "Please input a file to upload." and not "You have tried to upload file(s) that are not safe." liike your instructions say. Before applying your PR it was like descibed in your instructions.

File: administrator/components/com_media/controllers/file.php [hunk 1]
```diff
@@ -61,44 +61,48 @@ public function upload()
...
+			if (($postMaxSize > 0 && $contentLength > $postMaxSize)
+				|| ($memoryLimit != -1 && $contentLength > $memoryLimit))
+			{
+				// Files are too large.
+				JError::raiseWarning(100, JText::_('COM_MEDIA_ERROR_WARNUPLOADTOOLARGE'));
+
+				return false;
+			}
+
+			// No files wered provided.
```
Comment (Quy) [inline administrator/components/com_media/controllers/file.php]:
```suggestion
			// No files were provided.
```

**Comment by SharkyKZ** [comment]
@richard67 Should work now.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1c9e337c05bdc49c115df66671eecc1e84858207<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27730">issues.joomla.org/tracker/joomla-cms/27730</a>.</sub>

File: libraries/src/Input/Files.php [hunk 2]
```diff
@@ -85,7 +85,7 @@ public function get($name, $default = null, $filter = 'cmd')
 			);
 
 			// Prevent returning an unsafe file unless speciffically requested
```
Comment (Quy) [inline libraries/src/Input/Files.php]:
```suggestion
			// Prevent returning an unsafe file unless specifically requested
```

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 1c9e337c05bdc49c115df66671eecc1e84858207<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27730">issues.joomla.org/tracker/joomla-cms/27730</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27730">issues.joomla.org/tracker/joomla-cms/27730</a>.</sub>

**Pull request merged**